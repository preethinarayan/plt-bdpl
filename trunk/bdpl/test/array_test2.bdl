int i;
struct _a{
	struct _b
	{
		int[10] x;
		byte y;
		bit z;
	}b;
}[2] a;


for(i=0;i<10;i=i+1){
	a[0].b.x[i] = i;
print(a);
}
a[0].b.y = 3;
a[0].b.z = 1;

int n;

print("The values are: ");
for(i=0;i<10;i=i+1){
	print(a[0].b.x[i]);
	print(",");
}
print(a[0]);
print(a[1]);
		
