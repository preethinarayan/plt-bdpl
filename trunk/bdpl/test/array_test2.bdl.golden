ARRAY of struct:_a
[
]
